apps/emqx/src/emqx_tls_certfile_gc.erl

```diff
@@ -228,7 +228,8 @@ find_managed_files(Filter, Dir) ->
                         Acc
                 end;
             (AbsPath, {error, Reason}, Acc) ->
-                ?SLOG(notice, "filesystem_object_inaccessible", #{
+                ?SLOG(notice, #{
+                    msg => "filesystem_object_inaccessible",
```

Comment:
Ouch, thanks!

Generally speaking, I think missing certs directory is not an error, so adding a preceding clause like this should also help.
```
            (AbsPath, {error, enoent}, Acc) when AbsPath == Dir ->
                Acc;
```